test/test_multiprocessing.py

```diff
@@ -322,39 +333,67 @@ def test_inherit_tensor(self):
     @unittest.skipIf(NO_MULTIPROCESSING_SPAWN, "Disabled for environments that \
                      don't support multiprocessing with spawn start method")
     @unittest.skipIf(not TEST_CUDA_IPC, 'CUDA IPC not available')
-    def test_cuda(self):
+    def test_cuda_simple(self):
         torch.cuda.FloatTensor([1])  # initialize CUDA outside of leak checker
         self._test_sharing(mp.get_context('spawn'), torch.cuda.FloatTensor)
 
     @unittest.skipIf(NO_MULTIPROCESSING_SPAWN, "Disabled for environments that \
                      don't support multiprocessing with spawn start method")
     @unittest.skipIf(not TEST_CUDA_IPC, 'CUDA IPC not available')
-    @unittest.skipIf(not TEST_MULTIGPU, 'found only 1 GPU')
-    def test_cuda_small_tensors(self):
```

Comment:
This diff is hard to read. How did `test_cuda_small_tensors` change?

Reply:
Removed leak checker (it is checked in other test anyway). Hope changed indent will fix diff.